apps/docs/src/content/docs/guide/oauth.md

```diff
@@ -0,0 +1,95 @@
+
+The scope identifiers are currently best found in the backend source code located at [`apps/labrinth/src/models/v3/pats.rs`]. The scope parameter is an array of scope identifiers, seperated by a plus sign (`+`).
+
+The redirect URI is the endpoint on your server that will receive the code which can eventually be used to act on the user's behalf. For security reasons the redirect URI used has to be allowlisted in your application settings. The redirect will contain the following query parameters:
+
+| Query parameter | Description                                        |
+|-----------------|----------------------------------------------------|
+| `code`          | The code that can be exchanged for an access token |
+| `client_id`     | Your client id                                     |
+| `redirect_uri`  | The redirect URI which was used                    |
+| `grant_type`    | Always `authorization_code` in Modrinth            |
+
+## Exchanging tokens
+
+If you've followed the previous section on getting authorization, you should now have an authorization code. Before you can access the API, you need to exchange this code for an access token. This is done by sending a POST request to the exchange token endpoint, `https://api.modrinth.com/_internal/oauth/token`. This request has to be of type urlencoded form. Make sure the `Content-Type` header is set to `application/x-www-form-urlencoded`. To authenticate this request you need to place your client secret in the `Authorization` header.
+
+In the body use these fields:
+
+| Field          | Description                                                  |
+|----------------|--------------------------------------------------------------|
+| `code`         | The authorization code                                       |
+| `client_id`    | Your client id, the same as in the authorization request     |
+| `redirect_uri` | The redirect URI which was redirected to after authorization |
+| `grant_type`   | Always `authorization_code` in Modrinth                      |
+
+If the request succeeds, you should receive a JSON payload with these fields:
+
+| Field          | Description                                          |
+|----------------|------------------------------------------------------|
+| `access_token` | The access token you can use to access the API       |
+| `token_type`   | Currently only `Bearer`                              |
+| `expires_in`   | The amount of seconds until the access token expires |
+
+To use this access token, you attach it to API requests in the `Authorization` header. To get basic information about the authorizer, you can use the [`/user` endpoint], which automatically gets the user from the header.
+
+If you have any questions, you're welcome to ask in #api-development in the [Discord guild], or create an issue on [GitHub].
```

Comment:
One last thing, we would rather have people come to the [support website](https://support.modrinth.com/en/) first before making an issue. Otherwise LGTM